pages/welcome-kit.md

```diff
@@ -48,8 +49,8 @@ Based on where you are in life, your VA benefits and services can support you in
 
 Feel free to share this guide with friends or family members who need help with their benefits too. Download your VA Welcome Kit and print out copies for yourself and others:
 
-[Download your VA Welcome Kit in black and white](/WelcomeVA_Guide_print_version_final.pdf).
+[Download your VA Welcome Kit in black and white](/va-welcome-kit-b&w.pdf).
```

Comment:
I'm a little concerned about the `&` in the file name, but it does work in my Chrome and Firefox.

Reply:
good catch. I removed the `&`.